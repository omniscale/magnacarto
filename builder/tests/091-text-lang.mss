#labels {
    text-name: [name];
    text-fill: black;
    text-size: 8;
    text-lang: "[lang]";
}

