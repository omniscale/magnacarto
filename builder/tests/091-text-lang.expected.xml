<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="labels" filter-mode="first">
    <Rule>
      <TextSymbolizer fill="#000000" lang="[lang]" size="8">[name]</TextSymbolizer>
    </Rule>
  </Style>
  <Layer name="labels" srs="" status="off">
    <StyleName>labels</StyleName>
  </Layer>
</Map>
